PULL_REQUESTS.md

```diff
@@ -12,13 +12,21 @@ explaining the overall change. Both the component and the explanation must be lo
 * router:add x-envoy-overloaded header
 * tls: add support for specifying TLS session ticket keys
 
-### <a name="desc"></a>Description
+### <a name="desc"></a>Commit Message
 
-The description field should include a more verbose explanation of what this PR
-does. If this PR causes a change in behavior it should document the behavior
-before and after   If fixing a bug, please describe what the original issue is and
-how the change resolves it. If it is configuration controlled, it should note
-how the feature is enabled etc...
+The commit message field should include an explanation of what this PR
+does. This will be used as the final commit message that maintainers will use to
+populate the commit message when merging. If this PR causes a change in behavior
+it should document the behavior before and after. If fixing a bug, please
+describe what the original issue is and how the change resolves it. If it is
+configuration controlled, it should note how the feature is enabled etc...
+
+
+### <a name="desc"></a>Detailed Description
```

Comment:
Maybe this should be "Additional Description" to indicate that the meat of their description should belong in the commit message?

Reply:
Yeah makes sense to me.

Reply:
all set!